web/app/fr/gospeak/web/pages/orga/cfps/CfpForms.scala

```diff
@@ -13,5 +13,14 @@ object CfpForms {
     "end" -> optional(localDateTime),
     "description" -> markdown,
     "tags" -> tags
-  )(Cfp.Data.apply)(Cfp.Data.unapply))
-}
+  )(Cfp.Data.apply)(Cfp.Data.unapply) verifying("Start of Cfp should be anterior to its end", fields => fields match {
```

Comment:
so `data` is the same as `fields`, you could do `validate(_).isDefined` instead of pattern matching I think
Maybe calling the `validate` function `isStartBeforeEnd` and returning a Boolean can improve readability:
```scala
verifying("Start of Cfp should be anterior to its end" isStartBeforeEnd)

def isStartBeforeEnd(data: Cfp.Data): Boolean = ...
```